packages/gluestick/src/config/webpack/ChunksPlugin.js

```diff
@@ -22,17 +13,17 @@ const chunkInfoFilePath = (
   return path.join(webpackConfiguration.output.path, chunkInfoFilename);
 };
 
-const getChunksInfoBody = (json: Object, publicPath: string): ChunksInfo => {
-  const assetsByChunk: Object = json.assetsByChunkName;
+const getChunksInfoBody = (stats: Object, publicPath: string): ChunksInfo => {
```

Comment:
ChunksPlugin is from universal-webpack and was copy/pasted in here a while ago. It tracks JS + CSS chunks and writes the information to a file.